server/test/apiTest.js

```diff
@@ -68,28 +67,88 @@ describe("Testing api/neighborhoodList", () => {
     })
 })
 
-describe("Testing api/cards using router.delete", () => {
-    it("Should return successful delete message after deleting", () => {
-        let req = {
-            "id": "10"
+describe("Testing api/cards as a whole", () => {
+    it("Should have a larger list with our new item after adding", () => {
+        const newNeighborhood = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        chai.request("http://localhost:4000")
+            .post("/api/cards")
+            .send(newNeighborhood)
+            .end((err, res) => {
+                expect(err).to.be.null
+                expect(res).to.be.json
+            })
+
+        chai.request("http://localhost:4000")
+            .post("/api/neighborhoodList")
+            .send()
+            .end((err, res) => {
+                expect(err).to.be.null
+                expect(res).to.be.json
+                expect(res.body.length).to.be.equal(20641)
```

Comment:
It's possible this unit test (and similar ones that have a hardcoded length in them) will potentially have to be changed in the future if we end up saving to the original csv file. However, for now this seems like an alright approach.